### ISSUE TITLE
Doesn't actually work

### ISSUE THREAD

**Issue by jlnhlfan** [chrippa/livestreamer#1693]
So, I want to watch some Twitch streams, but can't because of constant buffering. I wanted to use this after finding out about it. Turns out, things aren't what they seem, because I got the following error when I tried to watch one of those streams

![Screenshot_March_11_2024_11_28_36_AM](https://github.com/chrippa/livestreamer/assets/31529087/4ebf0152-9af5-4d54-acd1-5815f491f551)

So until I get an answer as to what is causing this, I'm just going to stick to Twitch's copnstant buffering.

**Comment by Turtle-Boi** [comment]
I don't think this project is maintained anymore, you'll probably have to find an alternative.

**Comment by jlnhlfan** [comment]
Ah, okay.

**Comment by WolfganP** [comment]
https://github.com/streamlink/streamlink is the successor
